portality/static/widget/simple_search.js

```diff
@@ -37,9 +37,31 @@ function scriptLoadHandler() {
 
 /******** Our main function ********/
 function main() { 
-    jQuery(document).ready(function($) { 
```

Comment:
In this file, near the top we explicitly import an old version of jquery from a google cdn.  We should load this as the latest version from our domain if we need it.  Though I'm not actually convinced we need jquery here at all

Reply:
As discussed: we do jquery here.

Reply:
>We should load this as the latest version from our domain if we need it.

done 👍🏼